code/_onclick/hud/alert.dm

```diff
@@ -51,13 +51,17 @@
 	thealert.owner = src
 
 	if(new_master)
-		var/old_layer = new_master.layer
-		var/old_plane = new_master.plane
-		new_master.layer = FLOAT_LAYER
-		new_master.plane = FLOAT_PLANE
-		thealert.add_overlay(new_master)
-		new_master.layer = old_layer
-		new_master.plane = old_plane
+		// This should be made into a generic helper proc later tbh
+		var/mutable_appearance/master_appearance = new
+		master_appearance.appearance = new_master.appearance
```

Comment:
why not
```suggestion
		var/mutable_appearance/master_appearance = new(new_master)
```

Reply:
was too lazy to check docs to see if that's hot that worked